### DISCUSSION TITLE
React 17 support?

### DISCUSSION THREAD

**Discussion by TKDev7** [ariakit/ariakit#2231] [Questions] [answered]
Will this library support React 17?

**Comment by diegohaz** [comment, marked as answer]
Yes

**Reply by TKDev7** [reply]
I think it would be wise to not do so and just support v18

**Reply by diegohaz** [reply]
Why?

**Reply by TKDev7** [reply]
Because 18 is the latest version. I see no reason for this library to support an old version of react.
